overlord/snapstate/snapstate_update_test.go

```diff
@@ -1709,38 +1709,6 @@ func (s *snapmgrTestSuite) TestParallelInstanceUpdateRunThroughSkipBinaries(c *C
 	s.testParallelInstanceUpdateRunThrough(c, true)
 }
 
-func (s *snapmgrTestSuite) TestUpdateWithNewBase(c *C) {
```

Comment:
why is this one going away? or I'm reading the diff wrong?

Reply:
hm I don't remember doing this intentionally. It might've been a mistake I didn't notice because the PR was based on another branch. I'll restore it